YIF_Backend/Controllers/SuperAdminController.cs

```diff
@@ -329,5 +329,23 @@ public async Task<IActionResult> GetIoEAdminIdByIoEId(string ioEId)
             var result = await _superAdminService.GetIoEAdminIdByIoEId(ioEId);
             return Ok(result.Object);
         }
+
+        /// <summary>
+        /// Get Institution of Education information by Institution of Education id.
+        /// </summary>
+        /// <returns>List of moderators</returns>
+        /// <response code="200">Returns a list of moderators</response>
```

Comment:
Change code 200 description to "Returns information about Institution of Education"

Reply:
done